k8s/common/aac/identity.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
```

Comment:
this is in the wrong folder

Reply:
How so? I noticed across all other namespaces they had the identity.yaml file in the folder the way aac does.